source/documentation/common/security/proc-Applying_the_DISA_STIG_Security_Profile_RHV.adoc

```diff
@@ -7,42 +7,31 @@
 [id='applying-the-disa-stig-profile-rhvh_{context}']
 = Applying the DISA STIG profile in {enterprise-linux-shortname} based hosts and the standalone Manager
 
+When installing {virt-product-shortname}, you can select the DISA STIG profile with the UI installer, which is the profile provided by {enterprise-linux-shortname} 8.
```

Comment:
I just realized that this procedure (and the PCI-DSS procedure) are incorrect. You cannot do this while deploying the SHE. RHV does not have this installer UI. It's RHEL. 

The user accesses the UI by following the procedure in [Customizing installation](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/performing_a_standard_rhel_installation/graphical-installation_graphical-installation#doc-wrapper) while logged in to a RHEL host. 

The security profiles are applied in [Configuring a security profile](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/performing_a_standard_rhel_installation/graphical-installation_graphical-installation#configuring-security-policy_security-policy).

So you should link to the both sections and then add a note that the user must select the DISA STIG for RHEL 8 (probably it is the default; I'm not sure why it is not in the RHEL doc).